.github/workflows/integration.yml

```diff
@@ -83,6 +83,5 @@ jobs:
       - name: codecov
         uses: codecov/codecov-action@894ff025c7b54547a9a2a1e9f228beae737ad3c2 # 2.1.0
         with:
-         files: ./e2e-coverage.out
+         files: ./clients/localdir/e2e-coverage.out, ./clients/ossfuzz/e2e-coverage.out, ./clients/githubrepo/e2e-coverage.out, ./clients/gitlabrepo/e2e-coverage.out, ./clients/git/e2e-coverage.out, ./clients/e2e-coverage.out, ./cmd/e2e-coverage.out, ./attestor/policy/e2e-coverage.out, ./checker/e2e-coverage.out, ./checks/fileparser/e2e-coverage.out, ./checks/evaluation/e2e-coverage.out, ./checks/raw/e2e-coverage.out, ./checks/e2e-coverage.out, ./dependencydiff/e2e-coverage.out, ./e2e/e2e-coverage.out, ./e2e-coverage.out, ./cron/config/e2e-coverage.out, ./cron/internal/controller/e2e-coverage.out, ./cron/internal/format/e2e-coverage.out, ./cron/internal/pubsub/e2e-coverage.out, ./cron/internal/data/add/e2e-coverage.out, ./cron/worker/e2e-coverage.out, ./cron/data/e2e-coverage.out
```

Comment:
If I remember the test will fail if any of these files don't exist (refactor, test deleted etc), and it's a pain when our integration tests break.
Additionally, when we add new tests this will need to change as well. Is it worth trying a glob?

There's some indication that it could work at:
https://community.codecov.com/t/action-does-not-find-reports-by-glob-pattern/2110

Reply:
Something like this:
https://github.com/spencerschrock/scorecard-test/blob/4a67d801e41456f349a1291c2525621df6ac7952/.github/workflows/integration.yml#L69-L73

which picked up 23 files in the upload
https://github.com/spencerschrock/scorecard-test/actions/runs/4803332583/jobs/8547716157#step:8:65

Reply:
I agree. Thanks. I have updated the code.